src/utils/curriculum/features.test.ts

```diff
@@ -87,25 +65,8 @@ describe("isCurricPartnerHackEnabled", () => {
 });
 
 describe("getUnitFeatures", () => {
-  it("returns swimming rules when hack enabled and matching unit", () => {
```

Comment:
Do we need a replacement test for the new behaviour?

Reply:
I've added this back in now